wiki/en/Running-a-Server.md

```diff
@@ -150,21 +141,17 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
 
 <figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus server setup window"></figure>
 
-### Make My Server Public
-
-By default, you will be running a Private Server and need to [read these instructions](#running-a-private-server) to have others connect to you in this mode.
+### The Directory list 
 
-When making your server public, you should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.
+**None**: By default, you will not be connected to a directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your server in this mode.
 
+**Genre**: To allow other people to see your server on one of the built-in public directories, select your desired genre directory. You should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.  
 
-### Genre
-
-Because there is a limit of 150 servers per Directory, you need to select which Directory Server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). This will make your server appear in the relevant server list for clients.
-
+**Custom**: This sets your server to be a directory on which other servers can be listed. Note that when setting this mode, you may want also to specify the location of server list file (See "Options" below) unless you want to disconnect all listed servers when restarting Jamulus.
 
 ### My Server Info
 
-Type in the name, city and country so that other users can easily identify you.
+When running as a registered server this displays the server's name, city and country so that other users can easily identify it in the directory listing.
```

Comment:
> this displays

Not really the right way to say it.  This is the server set up screen.  It's not "displaying" it - it's allowing it to be set by the server operator.

Reply:
The fact they can be edited in the UI here is self-evident (it's a form) and is consistent with how we talk about My Profile profile too.

Reply:
But it's still wrong.  Is it okay to use bad English just because you use it all the time?

Reply:
It's standard practice to explain the _effect_ of using a UI, not _how_ you use it (unless your UI is unusable). So if you say "You can set the server's name, city and country... " then that begs the question why and how, which is the more important thing to explain.